docs/changelog.rst

```diff
@@ -3,6 +3,34 @@
 Changelog
 =========
 
+0.13.0 (Sep 2022)
+-----------------
+- CausalML surpassed `1MM downloads <https://pepy.tech/project/causalml>`_ on PyPI and `3,200 stars <https://github.com/uber/causalml/stargazers>`_ on GitHub. Thanks for choosing CausalML and supporting us on GitHub.
+- We have 7 new contributors @saiwing-yeung, @lixuan12315, @aldenrogers, @vincewu51, @AlkanSte, @enzoliao, and @alexander-pv. Thanks for your contributions!
+- @alexander-pv revamped `CausalTreeRegressor` and added `CausalRandomForestRegressor` with more seamless integration with `scikit-learn`'s Cython tree module. He also added integration with `shap` for causal tree/ random forest interpretation. Please check out the `example notebook <https://github.com/uber/causalml/blob/master/examples/causal_trees_interpretation.ipynb>`_.
+- We dropped the support for Python 3.6 and removed its test workflow.
+
+Updates
+~~~~~~~~~~~~~
+- Fix typo `(% -> $)` by @saiwing-yeung in https://github.com/uber/causalml/pull/488
+- Add function for calculating PNS bounds by @t-tte in https://github.com/uber/causalml/pull/482
+- Fix hard coding bug by @t-tte in https://github.com/uber/causalml/pull/492
+- Update README of `conda` install and instruction of maintain in conda-forge by @ppstacy in https://github.com/uber/causalml/pull/485
+- Update `examples.rst` by @lixuan12315 in https://github.com/uber/causalml/pull/496
+- Fix incorrect `effect_learner_objective` in `XGBRRegressor` by @jeongyoonlee in https://github.com/uber/causalml/pull/504
+- Fix Filter F doesn't work with latest `statsmodels`' F test f-value format by @paullo0106 in https://github.com/uber/causalml/pull/505
+- Exclude tests in `setup.py` by @aldenrogers in https://github.com/uber/causalml/pull/508
+- Enabling higher orders feature importance for F filter and LR filter by @zhenyuz0500 in https://github.com/uber/causalml/pull/509
+- Ate pretrain 0506 by @vincewu51 in https://github.com/uber/causalml/pull/511
+- Update `methodology.rst` by @AlkanSte in https://github.com/uber/causalml/pull/518
+- Fix the bug of incorrect result in qini for multiple models by @enzoliao in https://github.com/uber/causalml/pull/520
+- Test `get_qini()` by @enzoliao in https://github.com/uber/causalml/pull/523
+- Fixed typo in `uplift_trees_with_synthetic_data.ipynb` by @jroessler in https://github.com/uber/causalml/pull/531
+- Remove Python 3.6 test from workflows by @jeongyoonlee in https://github.com/uber/causalml/pull/535
+- Causal trees update by @alexander-pv in https://github.com/uber/causalml/pull/522
```

Comment:
@jpansnap it's mentioned here. Do you mean something else?

Reply:
oh i missed that one sorry!~